src/main/scala/se/lth/immun/TargetDeisotoper.scala

```diff
@@ -143,37 +238,28 @@ class TargetDeisotoper(val params:DinosaurParams) extends Timeable {
 			} else result
 		
 		val cleanResult =
-			if (z * seedTot.centerMz < 1000) {
+			if (z * seedCenterMz < 1000) {
 				val apex = oneMaxResult.maxBy(hills(_).total.intensity)
 				oneMaxResult.drop(oneMaxResult.indexOf(apex))
 			} else oneMaxResult
 		
 		val cleanProfile = cleanResult.map(hills(_).total.intensity)
-		val avgIsotopeDistr = Peptide.averagine((seedTot.centerMz - Constants.PROTON_WEIGHT)*z).getIsotopeDistribution()
+		val avgIsotopeDistr = Peptide.averagine((seedCenterMz - Constants.PROTON_WEIGHT)*z).getIsotopeDistribution()
 		
 		val alignment = params.adv.deisoAveragineStrategy(cleanProfile, avgIsotopeDistr, params)
 		
 		alignment.map(a => IsotopePatternInds(
-				cleanResult.drop(math.max(0, -a.offset)), 
-				math.max(0, a.offset), 
-				avgIsotopeDistr.intensities.indexOf(avgIsotopeDistr.intensities.max), 
+				cleanResult,//.drop(math.max(0, -a.offset)), 
+				0, //math.max(0, a.offset), 
+				0, //avgIsotopeDistr.intensities.indexOf(avgIsotopeDistr.intensities.max), 
```

Comment:
This seems like some debugging that was not reset, or what's going on here?

Reply:
I think this again had to do with Dinosaur sometimes reporting another isotope than the one we 'target'. We could use the same new flag as above to allow usage of the original functionality.

Reply:
As above, if this is more useful behavior, then remove the commented out code completely, and leave a short comment somewhere to explain why target mode works differently.